### DISCUSSION TITLE
Code Review IM-13

### DISCUSSION THREAD

**Discussion by AlinaDubchak** [HowProgrammingWorks/LiveQA#80] [Code review]
Пишемо браузерну гру "Тетріс"
Проект ще буде видозмінюватись, в планах додати показ наступної фігури та пришвидшення падіння при натисненні пробілу.
Наразі базовий функціонал забезпечений, проте потребує редагування.
Будемо вдячні за ревью [коду](https://github.com/AlinaDubchak/PROJECT.md)

Учасники проекту:
-Дубчак Аліна ([профіль](https://github.com/AlinaDubchak))
-Мартинюк Марія  ([профіль](https://github.com/mmarty12))

**Comment by tshemsedinov** [comment]
https://github.com/AlinaDubchak/PROJECT.md/issues/5

**Reply by AlinaDubchak** [reply]
made some [corrections](https://github.com/AlinaDubchak/PROJECT.md)
